packages/pharos/src/components/button/pharos-button.ts

```diff
@@ -299,8 +299,8 @@ export class PharosButton extends ScopedRegistryMixin(FocusMixin(AnchorElement))
             id="button-element"
             name="${ifDefined(this.name)}"
             value="${ifDefined(this.value)}"
-            ?autofocus=${this.autofocus}
-            ?disabled=${this.a11yDisabled ? false : this.disabled}
+            .autofocus=${this.autofocus}
+            .disabled=${this.a11yDisabled ? false : this.disabled}
```

Comment:
I'm curious on the change to move from [boolean attributes](https://lit.dev/docs/templates/expressions/#boolean-attribute-expressions) to [property expressions](https://lit.dev/docs/templates/expressions/#property-expressions)

From the docs I read 
> You can use the property expression syntax to pass complex data down the tree to subcomponents.

which might not apply so much for these attributes are boolean attributes on the underlying HTML elements?

Reply:
I reverted the changes in the components themselves, leaving only the changes in the stories templates. I did make a pass to convert all `some-attr="${this.someThing}"` to be `some-attr=${this.someThing}` for consistency and because quotes aren't necessary, and it also better visually distinguishes static string values from calculated ones. I don't think this should have any functional effect in practice, but reduces moving parts on this somewhat open question.